default/python/universe_generation/galaxy.py

```diff
@@ -109,15 +422,13 @@ def spiral_galaxy_calc_positions(positions, arms, size, width):
             break
 
         if not attempts:
-            print "Spiral galaxy shape: giving up on placing star", i,\
-                  ", can't find position sufficiently far from other systems"
+            print "Spiral galaxy shape: giving up on placing star {}, can't find position sufficiently far from other systems.".format(i)
```

Comment:
AFAIK the PEP8 max line length isn't 79/80, but 120. So that would be why the original statement was fine, while after your change it would have been just a bit too long. I'm not 100% sure about that however.